### ISSUE TITLE
Medusa automatically changing 'Scene Numbering' flag on American Dad!

### ISSUE THREAD

**Issue by NMJToolbox** [pymedusa/Medusa#8198]
Running Medusa on a Synology NAS.
Branch: masterCommit: d9c31da063e6c4b4383749e19b27b13693a1b71eVersion: 0.3.16
Database: 44.15
Python Version: | 2.7.14 (default, May 28 2018, 14:47:18) [GCC 4.7.3 20130102 (prerelease)]
SSL Version: | OpenSSL 1.0.2n 7 Dec 2017
OS: | Linux-3.10.105-x86_64-with-glibc2.2.5

Issue: American Dad! I have all of the episodes in numbering order of TheTVDB (currently up to season 17).  Medusa, even though when adding the show 'Scene numbering' is NOT turned on, Medusa turns it on and messes up the episode matching and subsequently thinks I already have episodes that have not even aired yet.

On initial adding the scene numbering option is off.
![image](https://user-images.githubusercontent.com/64314903/84920354-f213a200-b0ba-11ea-8532-75429e868191.png)

Add the show.
Before it starts scanning existing files I checked the options in the American Dad! page and scene numbering was off.
![image](https://user-images.githubusercontent.com/64314903/84920471-1b343280-b0bb-11ea-89ee-ee6a642f3435.png)

Whilst Medusa was scanning my files at some point it automatically changed the scene numbering option ON.
![image](https://user-images.githubusercontent.com/64314903/84920651-5898c000-b0bb-11ea-91af-d3113cb04b30.png)

Subsequently the episode matching is wrong.
![image](https://user-images.githubusercontent.com/64314903/84920727-77975200-b0bb-11ea-917d-b948286cc14e.png)

If I turn scene numbering OFF and force a rescan or full update, Medusa turns scene numbering back on.  There is no way I can find to turn it off and medusa not to revert it back.

I made backups of the database and checked and the scene numbering field in the DB is indeed set to 1 and not 0.  American Dad! appears to be the only TV show that it seems to do this with.

**Comment by mastafrooper** [comment]
I 'm having the same issue here.


Medusa Info: | Branch: masterCommit: d9c31da063e6c4b4383749e19b27b13693a1b71eVersion: 0.3.16Database: 44.15
-- | --
Python Version: | 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 22:22:05) [MSC v.1916 64 bit (AMD64)]
SSL Version: | OpenSSL 1.1.0j 20 Nov 2018
OS: | Windows-10-10.0.18362-SP0
Locale: | en_AU.cp1252

**Comment by p0psicles** [comment]
I'll check it out

**Comment by p0psicles** [comment]
@medariox 
https://github.com/pymedusa/Medusa/blob/master/medusa/show_queue.py#L644
What should we do with this?

**Comment by medariox** [comment]
I'd remove it. IMO it makes no sense to force this setting upon users.

**Comment by p0psicles** [comment]
@NMJToolbox sorry for taking so long to get back to you. This one got of my radar.
I'll remove the code that's repsonsible for forcing scene when scene numbering is found on thexem.de. But i'm curious about this:

> If I turn scene numbering OFF and force a rescan or full update, Medusa turns scene numbering back on. There is no way I can find to turn it off and medusa not to revert it back.

That should not be possible. The scene flag is only checked when adding a show AFAIK. So I can't reproduce this. @mastafrooper did you also notice this?

**Comment by NMJToolbox** [comment]
@p0psicles I had completely forgot about this but a few weeks after posting it stopped re-organising my episodes.  One thing I noticed is that scene numbering is now not forced on (no idea what I did to stop it doing it) but the XEM column is still ticked.  New episodes are being added properly now ie sticking to the TVDB numbering rather than forcing to scene numbering and my episodes are no longer being re-organised.
